src/scraper/scrapy_settings.py

```diff
@@ -0,0 +1,6 @@
+"""
+    Settings specific to scrapy
+"""
+
+# Settings passed to the crawler
+CRAWLER_SETTINGS = {"LOG_ENABLED": False}
```

Comment:
I'm glad to see that you guys are already considering configuration settings and at the same time keeping it simple!!!

I like that, please create a task for me when you get time to bring a configuration library into the project.
here the one I'm thinking to use: https://www.dynaconf.com
cc: @dieko95